compiler/luci/lang/include/luci/IR/Nodes/CircleTransposeConv.h

```diff
@@ -47,6 +47,8 @@ class CircleTransposeConv final
   loco::Node *outBackprop(void) const { return at(2)->node(); }
   void outBackprop(Node *node) { at(2)->node(node); }
 
+  // @note  "bias" of TransposeConv is only supported by circle node, not by tflite.
+  //        Q. In case of no bias, how is this filled? (Optional argument problem)
```

Comment:
@mhs4670go I want to complete this comment, but lack of information. Could I ask answer for this inline question? I would update this in actual PR

Reply:
cc @seanshpark

Reply:
@cgbahk When this circle file is loaded, optional argument that contains nothing - no bias - imports as `CircleOutputExlucde`. This node will be removed when it exports to circle file later.